Clicker/ViewControllers/CardController+Extension.swift

```diff
@@ -345,7 +345,15 @@ extension CardController: SocketDelegate {
         }
     }
 
-    func receivedFRFilter(_ pollFilter: PollFilter) { }
+    func receivedFRFilter(_ pollFilter: PollFilter) {
+        guard !pollFilter.success else { return }
+        let newPoll = Poll(poll: pollsDateModel.polls[currentIndex], state: pollsDateModel.polls[currentIndex].state)
+        newPoll.pollFilter = pollFilter
+        updateLatestPoll(with: newPoll)
+        adapter.performUpdates(animated: false, completion: nil)
+        let alertController = self.createAlert(title: "Inappropriate Content", message: "We have detected inappropriate language. Please submit an appropriate response. ")
```

Comment:
Is the last space at the end of the message intentional? Could we just make it "We have detected inappropriate language. Please submit an appropriate response."

Reply:
u rite - i just copied and pasted from zeplin but i'll update that